jib-maven-plugin/README.md

```diff
@@ -24,8 +24,16 @@ You can containerize your application easily with one command:
 mvn compile com.google.cloud.tools:jib-maven-plugin:0.9.1:build -Dimage=<MY IMAGE>
 ```
 
+This builds and pushes a container image for your application to a container registry.
+
 *If you encounter authentication issues, see [Authentication Methods](#authentication-methods).*
```

Comment:
This italicized part seems out of place in the middle of the section, maybe it should go at the end.

Reply:
Maybe this should just go on the same line as 27 since it is specifically in regards to pushing to a registry.

Reply:
Yeah, I was thinking about base image authentication but now I realize that isn't used in the quickstart.